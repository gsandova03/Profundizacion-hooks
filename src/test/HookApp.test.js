import React from 'react';
import { shallow } from 'enzyme';
import HooksApp from '../HooksApp';


describe('Preubas en el componente <HookApp/>', () => {
  test("debe de renderizar el componente correctamente", () => {
    const wrapper = shallow( <HooksApp /> )
    expect( wrapper ).toMatchSnapshot();
  })
})