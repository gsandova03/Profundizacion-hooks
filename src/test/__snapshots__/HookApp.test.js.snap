// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preubas en el componente <HookApp/> debe de renderizar el componente correctamente 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
